### ISSUE TITLE
data-catalog-frontend install issue

### ISSUE THREAD

**Issue by pnpierce713** [GetDKAN/dkan#3120]
When we are trying to run 'npm start' in step 5 from this guide: [https://github.com/GetDKAN/data-catalog-frontend](url) we are getting the following error:

` ERROR #11321  PLUGIN

"gatsby-node.js" threw an error while running the createPages lifecycle:

Request failed with status code 404

  Error: Request failed with status code 404

  - createError.js:16 createError
    [data-catalog-frontend]/[axios]/lib/core/createError.js:16:15

  - settle.js:17 settle
    [data-catalog-frontend]/[axios]/lib/core/settle.js:17:12

  - http.js:236 IncomingMessage.handleStreamEnd
    [data-catalog-frontend]/[axios]/lib/adapters/http.js:236:11

  - task_queues.js:84 processTicksAndRejections
    internal/process/task_queues.js:84:21

  - From previous event:

  - api-runner-node.js:258 runAPI
    [data-catalog-frontend]/[gatsby]/dist/utils/api-runner-node.js:258:22

  - api-runner-node.js:375 Promise.catch.decorateEvent.pluginName
    [data-catalog-frontend]/[gatsby]/dist/utils/api-runner-node.js:375:15

  - From previous event:

  - api-runner-node.js:374
    [data-catalog-frontend]/[gatsby]/dist/utils/api-runner-node.js:374:12

  - timers.js:456 processImmediate
    internal/timers.js:456:21

  - From previous event:

  - api-runner-node.js:368
    [data-catalog-frontend]/[gatsby]/dist/utils/api-runner-node.js:368:11

  - From previous event:

  - api-runner-node.js:275 module.exports
    [data-catalog-frontend]/[gatsby]/dist/utils/api-runner-node.js:275:12

  - index.js:421 module.exports
    [data-catalog-frontend]/[gatsby]/dist/bootstrap/index.js:421:9

  - develop-process.js:433 async module.exports
    [data-catalog-frontend]/[gatsby]/dist/commands/develop-process.js:433:7

failed createPages - 0.095s
`

Is anyone familiar with this error and what is happening? Is there a way to tell what the URL of the page that's throwing the 404 error is?

**Comment by janette** [comment]
@pnpierce713 did you check the endpoint I mentioned in the [last ticket](https://github.com/GetDKAN/dkan/issues/3117#issuecomment-639071986)? do you have datasets listed there?

**Comment by pnpierce713** [comment]
We ended up not using our own Drupal site and used the dkan-docker images and now we have dkan running. The confusion started from [https://demo.getdkan.com/modules/contrib/dkan/docs/installation.html](url) in which we installed dkan as described on that page and when we went to the 'data catalog frontend' section it says to follow the dkan-tools readme to setup the backend. I now see there is a --demo-backend flag but it wasn't clear what to do on our first install. We also experienced a problem as in RHEL/CentOS 7 the version of docker is 1.13 when docker 19 is the latest variant which also caused issues when starting those containers. We were also running Apache on port 80 as the link from the getdkan site had us set that as a prerequisite. A disclaimer that if you were linked from getdkan that you want to stop apache may help. Another helpful section would be for Red Hat/CentOS users to install the docker-ce yum repository like this:

`wget https://download.docker.com/linux/centos/docker-ce.repo -O /etc/yum.repos.d/docker-ce.repo
yum -y install docker-ce`
`systemctl start docker`
`systemctl enable docker`
